marlowe-playground-client/src/MainFrame/Types.purs

```diff
@@ -259,6 +290,9 @@ _hasUnsavedChanges' = prop (SProxy :: SProxy "hasUnsavedChanges")
 _hasUnsavedChanges :: Lens' State Boolean
 _hasUnsavedChanges = _Newtype <<< _hasUnsavedChanges'
 
+_workflow :: Lens' State (Maybe Lang)
+_workflow = _Newtype <<< prop (SProxy :: SProxy "workflow")
```

Comment:
refactoring note (not for this PR). Do we still need to newtype the mainframe state?

Reply:
mhmhmh, not sure why it was added in the first place 🧐.
Maybe we can revisit this when we change `settings` to be a `MonadAsk`